src/_locales/en_US/messages.json

```diff
@@ -359,6 +359,22 @@
         "message": "Privacy Policy",
         "description": "Shown at the bottom of the intro page."
     },
+    "intro_pin_nudge": {
+        "message": "$START_SPAN_TAG$First, pin Privacy Badger to your toolbar$END_SPAN_TAG$ for easy access to the \"$DISABLE_FOR_SITE$\" button.",
```

Comment:
Should "First," be bolded? Not sure, just wondering which one looks/reads better.

Reply:
I originally thought it looked better with "First" bolded but now I think the opposite! What do you think? I don't have a strong preference

<img width="1195" alt="Screenshot 2024-07-01 at 11 13 34 AM" src="https://github.com/EFForg/privacybadger/assets/40436018/e3126ad0-4194-46d8-8419-9e4e8072013a">
<img width="1203" alt="Screenshot 2024-07-01 at 11 14 04 AM" src="https://github.com/EFForg/privacybadger/assets/40436018/ddcae076-c0d9-404e-9e0f-97f7bef1d0da">

Reply:
Same ...